web/tests/composebox_typeahead.test.js

```diff
@@ -28,6 +28,9 @@ const input_pill = mock_esm("../src/input_pill");
 const message_user_ids = mock_esm("../src/message_user_ids", {
     user_ids: () => [],
 });
+mock_esm("../src/settings_data", {
+    user_can_access_all_other_users: () => true,
```

Comment:
These changes are needed since previously `current_user.user_id` was mostly undefined and the function just returned true.